pandas/io/sas/sas7bdat.py

```diff
@@ -255,7 +253,8 @@ def _get_properties(self) -> None:
         buf = self._read_bytes(const.align_2_offset, const.align_2_length)
         if buf == const.align_1_checker_value:
             align1 = const.align_2_value
-        total_align = align1 + align2
+        else:
+            align1 = 0
```

Comment:
Nit: Might as well rename to `align`?

Reply:
This name follows what's in `const`, we could change it there but I suggest we just leave it like it is